### PR TITLE
Remake of 13754 - (...) Slips dont stop tail wagging

### PR DESCRIPTION
**Pull request by Sniblet**
At last.

# Document the changes in your pull request

Jamie's #13754 of #13452 doesn't do anything, I'm going to try and do something modular once someone like @nmajask tells me what that means
The end goal is that slipping will not stop tails wagging

- [x] tested
- [x] works

# Changelog

:cl:  
bugfix: Slips no longer stop tails wagging
/:cl:
And this time it really is a bugfix

### CODE REVIEW

**Comment by Cartlord** [comment]
Why do you keep trying to do this?

**Comment by Sniblet** [comment]
It rids the game of an inconvenience that subtly but consistently worsens my experience while improving no one’s, I have free time, no one else cares enough to do it, and no one has given a respectable (the _only_ counterargument I’ve read is “slipping should probably stop tail wagging”) reason not to
It would be in already if there hadn’t been a sudden need for modularity, or if Jamie got his version right, or possibly if the original PR with stuns-as-opposed-to-paralyzes not stopping tail wagging hadn’t been drowned in insubstantial, dispassionate, two-sentence “this is bad/your reasoning is bad/I literally just don’t get it (despite you writing paragraphs of justification and clearly being ready to give more if I would just ask you)” comments
I’m not fighting the power here, just wrestling the code for two months straight while the public generally sits back and chuckles at the code illiterate cat main’s pain and occasionally changes the rules I have to work under and always disapproves of my work because, here I can only put words in your mouth because you won’t speak, “muh furries”
I sound frustrated because I am and I have been for two months

**Comment by ghost** [comment]
No

**Comment by Sniblet** [comment]
I have full cause of weeping, but this heart
Shall break into a hundred thousand flaws
Or ere I’ll weep. O fool, I shall go mad!

**Comment by Moggram** [comment]
> It rids the game of an inconvenience that subtly but consistently worsens my experience while improving no one’s, I have free time, no one else cares enough to do it, and no one has given a respectable (the _only_ counterargument I’ve read is “slipping should probably stop tail wagging”) reason not to It would be in already if there hadn’t been a sudden need for modularity, or if Jamie got his version right, or possibly if the original PR with stuns-as-opposed-to-paralyzes not stopping tail wagging hadn’t been drowned in insubstantial, dispassionate, two-sentence “this is bad/your reasoning is bad/I literally just don’t get it (despite you writing paragraphs of justification and clearly being ready to give more if I would just ask you)” comments I’m not fighting the power here, just wrestling the code for two months straight while the public generally sits back and chuckles at the code illiterate cat main’s pain and occasionally changes the rules I have to work under and always disapproves of my work because, here I can only put words in your mouth because you won’t speak, “muh furries” I sound frustrated because I am and I have been for two months

You want 'reasoning' then you will have it. If we look at this logically, when cat humies slip they should land on their stomachs as a cat would. This would stop your tail from wagging because it would need to be used as a counterbalance weights as you fall. Also the sudden impact from hitting the floor would jolt the nervous system and stop 'tall wagging'. If you have been on the internet long enough, you 'Know what types of videos I am referring to. Looks like you are making the wrong RP and need put a end to this madness.

And that is why you are a git.

**Comment by Sniblet** [comment]
for the full experience including music please view as a slideshow (press ctrl and f5 together for you boomers)
https://docs.google.com/presentation/d/15fpCbEhl3eO8D_NJ9zrmjHKZaxiKZ6vI8O7FHuEGi-I/edit?usp=sharing

**Comment by Moggram** [comment]
> for the full experience including music please view as a slideshow (press ctrl and f5 together for you boomers) https://docs.google.com/presentation/d/15fpCbEhl3eO8D_NJ9zrmjHKZaxiKZ6vI8O7FHuEGi-I/edit?usp=sharing



Yur still a git.

**Comment by Hopekz** [comment]
> for the full experience including music please view as a slideshow (press ctrl and f5 together for you boomers) https://docs.google.com/presentation/d/15fpCbEhl3eO8D_NJ9zrmjHKZaxiKZ6vI8O7FHuEGi-I/edit?usp=sharing

neat
PR's normally don't come with a manifesto

**Comment by Sniblet** [comment]
Currently works
I have it on authority that those dots make this modular
If this isn't modular again then please tell me what modular means and how to make it modular instead of closing the PR and replacing it with one that doesn't work
Or close it and replace it with one that works
so that this doesn't go on another week

Also, glad you like it Hopekz - don't forget that most PRs also don't quote King Lear

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -530,17 +530,13 @@
...
-			wagging = TRUE
 		if(!(lube & SLIDE_ICE))
 			C.Knockdown(knockdown_amount)
 			C.Stun(stun_amount)
 			C.stop_pulling()
 		else
 			C.Knockdown(20)
-		if(wagging)
-			C.dna.species.start_wagging_tail(src)
+
```
Comment (JamieD1) [inline code/game/turfs/open.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**